src/node/buffer.ts

```diff
@@ -19,6 +19,8 @@ const atob = globalThis.atob;
 const btoa = globalThis.btoa;
 // eslint-disable-next-line @typescript-eslint/no-unsafe-assignment
 const Blob = globalThis.Blob;
+// eslint-disable-next-line @typescript-eslint/no-unsafe-assignment
```

Comment:
nit: disable eslint for the [whole section](https://eslint.org/docs/latest/use/configure/rules#disabling-rules) instead